src/main/java/com/tiph/dysonsphereproject/common/blocks/SolarGenerator.java

```diff
@@ -3,36 +3,30 @@
 import net.minecraft.core.BlockPos;
 import net.minecraft.world.level.BlockGetter;
 import net.minecraft.world.level.block.Block;
-import net.minecraft.world.level.block.SoundType;
-import net.minecraft.world.level.block.state.BlockBehaviour;
 import net.minecraft.world.level.block.state.BlockState;
 import net.minecraft.world.phys.shapes.CollisionContext;
 import net.minecraft.world.phys.shapes.VoxelShape;
-import net.neoforged.neoforge.registries.DeferredBlock;
-
-public class SolarGeneratorBlock extends DysonBlock {
 
+public class SolarGenerator extends Block {
```

Comment:
should this implement IResource?

Reply:
IDK i want it to but that getter needs to be static and the interface doesn't like that i don't know java